impl/src/main/java/com/okta/sdk/impl/ds/DefaultDataStore.java

```diff
@@ -467,17 +483,28 @@ private Response execute(Request request) throws ResourceException {
 
         Assert.notNull(response, "response argument cannot be null.");
 
-        Map<String, Object> out = null;
+        if (response.hasBody() && (MediaType.APPLICATION_JSON.equals(response.getHeaders().getContentType())
+            || APPLICATION_ION_JSON.equals(response.getHeaders().getContentType()))) {
+            return mapMarshaller.unmarshal(response.getBody(), response.getHeaders().getLinkMap());
+        }
+
+        return java.util.Collections.emptyMap();
+    }
+
+    private Optional<InputStream> getRawBody(Response response) {
+        Assert.notNull(response, "response argument cannot be null.");
 
         if (response.hasBody()) {
-            out = mapMarshaller.unmarshal(response.getBody(), response.getHeaders().getLinkMap());
+            return Optional.ofNullable(response.getBody());
         }
```

Comment:
this reads funny, if response has body, then Optional.ofNullable

Why not use `Optional.of(value)` instead, or skip the if?